scripts/auto_restart_devnet.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -e
+
+pushd $(dirname "$0")/..
+
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "** Interrupted by CTRL-C"
+    DEVNET_PID=$(pgrep devnet)
```

Comment:
should we instead save pid when starting devnet?
https://unix.stackexchange.com/questions/30370/how-to-get-the-pid-of-the-last-executed-command-in-shell-script

Reply:
good point